scripts/travis/install.sh

```diff
@@ -12,6 +12,32 @@ set -e
     travis_time_end
 )
 
+# ROLL A FEW R PACKAGES BACK TO SPECIFIED VERSIONS
+(
+    travis_time_start "pecan_install_dbplyr" "Installing dbplyr version 1.3.0 see #2349"
+    Rscript -e 'devtools::install_version("dbplyr", version = "1.3.0", repos = "http://cran.us.r-project.org")'
```

Comment:
I think this has been resolved by #2392. Does the build succeed if you remove it?

Reply:
Yeah, best I can tell it does. Removed in 123a50ac9, description update momentarily.